docs/api-reference/geo-layers/h3-hexagon-layer.md

```diff
@@ -103,6 +103,38 @@ Possible values:
 * `false`: The layer chooses the mode automatically. High-precision rendering is only used if an edge case is encountered in the data.
 * `true`: always use high-precision rendering.
 
+##### `lowPrecision` (Boolean, optional)
```

Comment:
It doesn't make sense to have two props for controlling the same thing. What if `highPrecision` and `lowPrecision` are both true? Can you just make `highPrecision` accept `true|false|'auto'` and default to `'auto'`?

Reply:
Is it going to be a breaking change in such case? As atm false indicates that precision should be selected automatically.

Reply:
Yes it will be breaking. Still better than adding confusing API IMO